tests/Oryx.BuildImage.Tests/DotNetCoreSampleAppsTest.cs

```diff
@@ -25,7 +25,7 @@ public DotNetCoreSampleAppsTest(ITestOutputHelper output) : base(output)
         }
 
         private DockerVolume CreateSampleAppVolume(string sampleAppName) =>
-            DockerVolume.CreateMirror(Path.Combine(_hostSamplesDir, "DotNetCore", sampleAppName));
+            DockerVolume.Create(Path.Combine(_hostSamplesDir, "DotNetCore", sampleAppName));
```

Comment:
Can you introduce this change in a separate PR, so we can clear out all these changed files from this PR?

Reply:
Sure, sounds good